pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildPexTask.java

```diff
@@ -121,6 +125,9 @@ private void configureExecution(PythonExtension pythonExtension, ExecSpec spec)
             wheelExtension.getWheelCache().getAbsolutePath(),
             "--no-deps",
             ".");
+        spec.setErrorOutput(outputStream);
+        spec.setErrorOutput(outputStream);
```

Comment:
Should one of these two lines be `setStandardOutput` instead of both setting error output?

Reply:
Yep. Fixed.